Frameworks/Foundation/NSCache.mm

```diff
@@ -286,7 +286,7 @@ - (void)_evictEntriesOverCharge {
 
     auto rit = _cacheEntries.end();
     do {
-        _NSCacheEntry* entry = (id)*--rit;
+        _NSCacheEntry* entry = (id) * --rit;
```

Comment:
I actually reverted quite a few already, the ones remaining are ones that I thought were good for clang to do. I can still revert them if you feel that that's the proper course.

---
In reply to: [72162462](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72162462) [](ancestors = 72162462,70353130,70347524)